engine/apps/phone_notifications/models/phone_call.py

```diff
@@ -0,0 +1,55 @@
+from django.db import models
+
+
+# Duplicate to avoid circular import to provide values for status field
+class TwilioCallStatuses:
+    QUEUED = 10
+    RINGING = 20
+    IN_PROGRESS = 30
+    COMPLETED = 40
+    BUSY = 50
+    FAILED = 60
+    NO_ANSWER = 70
+    CANCELED = 80
+
+    CHOICES = (
+        (QUEUED, "queued"),
+        (RINGING, "ringing"),
+        (IN_PROGRESS, "in-progress"),
+        (COMPLETED, "completed"),
+        (BUSY, "busy"),
+        (FAILED, "failed"),
+        (NO_ANSWER, "no-answer"),
+        (CANCELED, "canceled"),
+    )
+
+
+class OnCallPhoneCall(models.Model):
```

Comment:
I tried just PhoneCall name, but it doesn't works
For example we have this method in ProveProvider interface. If I rename OnCallPhoneCall to PhoneCall, then I should rename oncall_phone_call parameter name to phone_call which makes no sense in function singanure. On the other side I don't like PhoneCall model name, but oncall_phone_call parameter, since it is confusing.
```
def make_notification_call(self, number, message, oncall_phone_call):
        ....
```
Same issue we have with TwilioPhoneCall model. Is has field oncall_phone_call. Renaming it to just phone call leads to lines like ```twilio_phone_call.phone_call```. 
I'm not 100% happy with OnCallPhoneCall name, but having just PhoneCall introduces more confusion.

Reply:
@vadimkerr what do you think about PhoneCallRecord instead of OnCallPhoneCall. Also it sounds well: twilio_phone_call.phone_call_record ( at least better that twilio_phone_call.oncall_phone_call)